apps/block_scout_web/lib/block_scout_web/views/api/rpc/epoch_view.ex

```diff
@@ -0,0 +1,55 @@
+defmodule BlockScoutWeb.API.RPC.EpochView do
+  use BlockScoutWeb, :view
+
+  alias BlockScoutWeb.API.RPC.RPCView
+
+  alias Explorer.Chain.Wei
+
+  def render(json, %{rewards: rewards}) when json in ~w(getvoterrewards.json) do
+    prepared_rewards = prepare_voter_rewards(rewards)
+
+    RPCView.render("show.json", data: prepared_rewards)
+  end
+
+  def render("error.json", assigns) do
+    RPCView.render("error.json", assigns)
+  end
+
+  defp prepare_voter_rewards(rewards) do
+    %{
+      total: %{
+        celo: to_string(rewards.total_amount |> Wei.to(:ether)),
+        wei: to_string(rewards.total_amount)
```

Comment:
Extract this transformation as a separate function?

Reply:
agreed, something like

```
defp format_celo_and_wei(value) do
  %{ 
      celo: value |> Wei.to(:ether)|> to_string(),
        wei: value |> to_string()
   }   
end
```

and then call this as necessary to construct the values